src/pages/JoinTheTeam/JoinTheTeam.css

```
@@ -0,0 +1,216 @@
+
+body{
+    max-width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin: 0%;
+}
+
+.Join {
+    background-color: #FAFAFA;
+    max-width: 100%;
+    width: 100%;
+    position: relative;
+    color: #000000;
+    font-family: 'Source Sans Pro', sans-serif;
+    padding-bottom: -2000px;
+}
+
+.Join p + p,
+.Join ul + p {
+    margin-top: 12px;
+}
+
+section {
+    margin: 2em 0;
+}
+
+.Join .join-header {
+    position: relative;
+    top: 3.68%;
+    padding-top: 130px;
+    font-family: 'Open Sans', sans-serif;
+    max-width: 100%;
+}
+
+.Join .join-header * {
+    text-align: center;
+}
+
+.Join .join-sub-header {
```

Just one more hehe :)

```suggestion
.join-sub-header {
```